client/src/components/GetDetails/index.js

```diff
@@ -1,21 +1,29 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Title from '../Shared/Title';
 import Header from '../Shared/Header';
 import Form from '../Shared/Form';
 import GButton from '../Shared/GreenButton';
 import Button from '../Shared/Button';
 import Footer from '../Shared/Footer';
 
-function GetDetails() {
-  return (
-    <React.Fragment>
-      <Title />
-      <Header />
-      <Form />
-      <Button />
-      <GButton />
-      <Footer />
-    </React.Fragment>
-  );
+class GetDetails extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const image = this.props.location.img;
+    return (
+      <React.Fragment>
+        <Title />
+        <Header title="Get your details" />
+        <Form image={image} />
```

Comment:
Have a look in your indentation here to make it clear which elements are children.

Reply:
it's good with eslint